scripts/libcbor_customize/configuration.h

```diff
@@ -0,0 +1,38 @@
+#ifndef LIBCBOR_CONFIGURATION_H
```

Comment:
Add a comment at the top of this file like: 
```
/**
 * DO NOT DIRECTLY MODIFY THIS FILE:
 *
 * The code in this file is generated from scripts/.../path_to_actual_source
 *  and any modifications should be in there.
 */
```

Otherwise people are going to tweak the comiler warnings below to fix some random CI failure. Then a few months later it will get blown away when someone updates cbor using this script